foursight_core/react/api/react_api_base.py

```diff
@@ -86,10 +86,14 @@ def create_forbidden_response() -> Response:
         return ReactApiBase.create_response(http_status=403, body={"status": "Forbidden."})
 
     @staticmethod
-    def create_error_response(message: Union[str, Exception]) -> Response:
+    def create_error_response(message: Union[dict, str, Exception]) -> Response:
         if isinstance(message, Exception):
             message = get_error_message(message)
-        return ReactApiBase.create_response(http_status=500, body={"error": message})
+        if isinstance(message, str):
+            body = { "error": message }
+        else:
+            body = message
```

Comment:
I'd probably put a blank line here to emphasize you're starting a new conditional. I find the control structure here needlessly hard to read.

If you want the 'cascading repair pattern' (I'm making up a name, but it's done a lot), maybe also write this as follows, but only use this if you think it makes it clearer. Note I changed what variable gets assigned in the consequent of this `if` so that it would align with the expectations 
```suggestion

        if isinstance(message, str):
            # Wrap a string message into a dictionary 
            message = { "error": message }

        body = message
```

ALTERNATIVELY, it might work better to just do the entire body as the following (untested). I find this format a LOT easier to read. Maybe you do, too?
```
if isinstance(message, Exception):
    return {"error": get_error_message(message)}
elif isinstance(message, str):
    return {"error": message}
elif isinstance(message, dict):
    return message
else:
    raise ValueError(f"The message argument must be a dict, str, or Exception: {message!r}")
```

Reply:
Yes, I do, thanks, will go with something like this.